lib/image_composition.dart

```diff
@@ -39,6 +39,9 @@ class _Composed {
 }
 
 /// The [ImageComposition] allows for composing multiple images onto a single image.
+/// 
+/// **Note:** Composing images is a heavy async operation and should not be called on each [Game.render].
+
```

Comment:
nit extra empty line (join doc to the class)

Reply:
Woepsiedoepsie